phpunit.xml.dist

```diff
@@ -27,6 +27,10 @@
         <!-- Enable this if you have installed ZendService\ReCaptcha on the
              include_path or via Composer. -->
         <env name="TESTS_ZEND_CAPTCHA_RECAPTCHA_SUPPORT" value="false" />
+        <!-- Change these if you want to use your own keys -->
```

Comment:
Are these real values, or mocked up ones? If we're using real values, these should be encrypted in the travis-ci config, or else somebody else may abuse them (unless google has "test values")